spec/support/saml_auth_helper.rb

```diff
@@ -105,6 +105,7 @@ def sp1_saml_settings
 
   def aal3_sp1_saml_settings
     settings = saml_settings.dup
+    settings.authn_context = nil
```

Comment:
should this be something that sets AAL3? Or does it come from the issuer?

Reply:
Yes, this is a case where the semantics have changed. The default settings request an AAL2 auth, and would have returned AAL3 given `aal: 3`. Now that request would return AAL2 since the request trumps the default. These settings are now used in a test of the default (spec/features/saml/aal3_required_spec), so we need to not make an AAL request at all.